packages/shorebird_cli/lib/src/archive_analysis/archive_differ.dart

```diff
@@ -16,6 +16,7 @@ abstract class ArchiveDiffer {
     'AssetManifest.bin',
     'AssetManifest.json',
     'NOTICES.Z',
+    'shorebird.yaml',
```

Comment:
This is kinda a big hammer.  You might want to just make the patch shorebird.yaml look like the release on instead of doing this?